content/labs/intro-to-ada/chapters/records.rst

```diff
@@ -415,7 +414,7 @@ keep track of assets.
     #. Procedure :ada:`Add` adds an item to the assets.
 
         #. Since we want to keep track of the assets, the implementation must
-           accumulate the total amount of each item.
+           accumulate the total value of each item's inventory, the result of multiplying the item quantity and its price.
```

Comment:
```suggestion
           accumulate the total value of each item's inventory, the result of 
           multiplying the item quantity and its price.
```

Reply:
Just adapting to the 80-character formatting... ;-)

Reply:
ahhh I'm sorry, I need to get better at that!